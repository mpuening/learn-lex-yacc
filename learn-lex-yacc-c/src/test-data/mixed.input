a = 15*2*(6*2+3)+4;
b = (1+2)*3*4;
c = (((1.0+2)))*((3.0*4));
d = 1+2*3-4/2;
e = 1*2+4/2-2;
f = 1+2+3-4+5*3-4/2;
g = 1*2+3-4-5+4/2-2;