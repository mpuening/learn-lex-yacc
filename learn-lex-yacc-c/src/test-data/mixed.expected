a = 454.000000
b = 36.000000
c = 36.000000
d = 5.000000
e = 2.000000
f = 15.000000
g = -4.000000
